src/Symfony/Component/VarDumper/Caster/ExceptionCaster.php

```diff
@@ -24,23 +24,6 @@ class ExceptionCaster
 {
     public static $srcContext = 1;
     public static $traceArgs = true;
-    public static $errorTypes = array(
```

Comment:
Removing a public property is a BC break. This must be kept (but deprecating it)